### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
fixed accordions in overview and release notes, FAQs updated, side bar updated, step 2 included in the heading.

### CODE REVIEW

File: overview.md
Comment (eu-tee) [inline overview.md:1]: @SenthilKumarSivakami still got inconsistencies on the <space>() vs <no_space>(), can i trouble u to do a sweep thru all these documents that u are going to push out?

File: overview.md [hunk 1]
```diff
@@ -51,22 +39,13 @@ Device management layer of SEED. It establishes a robust security baseline autom
```
Comment (eu-tee) [inline overview.md:39]: DEEP is still mentioned here.

File: release-notes.md
Comment (eu-tee) [inline release-notes.md:1]: Can i trouble you to make sure if the first word of the title needs to be capitalised, for consistency?

Comment (eu-tee) [inline release-notes.md:1]: there is also some scrolling issue with the right bar, it doesnt match what i am seeing for the content

File: _sidebar.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
-**Overview**
+- **Overview**
   - [Overview](overview)
```
Comment (eu-tee) [inline _sidebar.md:2]: This overview online has 1 item, need a expand/collapse?

File: _sidebar.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+- **Getting started**  
   - [Step 0: Prerequisites](prerequisites-for-onboarding)
   - [Step 1: Identify persona](identify-seed-onboarding-persona)
   - [Step 2: Onboard device](onboard-device/onboard-device-to-seed)
-**After onboarding**    
+- **After onboarding**    
   - [Post onboarding instructions](post-onboarding-instructions/post-onboarding-steps-and-verification)
   - [Device clean-up policy](device-clean-up-policy)
-  **Offboard device**
+- **Offboard device**
```
Comment (eu-tee) [inline _sidebar.md:13]: This offboard has 1 item, need a expand/collapse?

File: _sidebar.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
   - [Offboard device from SEED](offboard-device/offboard-device-from-seed)
-**Troubleshoot**
+- **Troubleshoot**
```
Comment (eu-tee) [inline _sidebar.md:15]: this one also only 1

**Review by eu-tee — APPROVED** [review]

File: overview.md
Comment (SenthilKumarSivakami) [reply overview.md:1]: fixed

File: overview.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply overview.md:39]: fixed.

File: release-notes.md
Comment (SenthilKumarSivakami) [reply release-notes.md:1]: In the title "Release notes" the first letter "R" is capitalised.  That is what we are following.

**Pull request merged**